pkg/flare/config_check_test.go

```diff
@@ -0,0 +1,92 @@
+package flare
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/DataDog/datadog-agent/pkg/autodiscovery/integration"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+type ConfigType string
+
+const (
+	MetricsConfig ConfigType = "metrics"
+	LogsConfig    ConfigType = "logs"
+)
```

Comment:
I don't think we need these to be public.

Reply:
You are right. Changed :+1: